frontend/Angular/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM node:11.6.0-alpine AS builder
+COPY . ./Angular
+WORKDIR /Angular
+RUN npm i
+RUN $(npm bin)/ng build --prod
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Quote this to prevent word splitting.](https://app.codacy.com/app/iamprovidence/CVRPWT/pullRequest?prid=4594462)